parlai/core/logs.py

```diff
@@ -94,3 +96,102 @@ def log_metrics(self, setting, step, report):
 
     def flush(self):
         self.writer.flush()
+
+
+class WandbLogger(object):
+    """
+    Log objects to Weights and Biases.
+    """
+
+    @classmethod
+    def add_cmdline_args(
+        cls, parser: ParlaiParser, partial_opt: Optional[Opt] = None
+    ) -> ParlaiParser:
+        """
+        Add w&b CLI args.
+        """
+        logger = parser.add_argument_group('Tensorboard Arguments')
+        logger.add_argument(
+            '-wblog',
+            '--wandb-log',
+            type='bool',
+            default=False,
+            help="Enable W&B logging of metrics, default is %(default)s",
```

Comment:
why is default false here? LOL maybe same question, why is `tensorboard_log` default `False`

Reply:
i'm good with keeping it `False`, since it requires additional overhead/setup that is not necessary to just train a model